projects/app-host/src/app/pages/dynamic/dynamic.component.ts

```diff
@@ -56,6 +56,10 @@ export class DynamicComponent implements OnInit {
     this.iotEnsCtxt.ToggleEmulatedEnabled();
   }
 
+  public UpdateRefreshRate(event: number){
+    this.iotEnsCtxt.UpdateTelemetrySync(event, this.State.Telemetry.PageSize);
```

Comment:
Add call to:
    this.State.Loading = true;

Before the  call to UpdateTelemetrySync.  The toggle methods don't set that because they manage their loading state differently, but the UpdateTelemetrySync should be more like Enroll Device.